src/store/modules/imageUploadStore.js

```diff
@@ -0,0 +1,33 @@
+import UploadService from '../../services/UploadService'
+
+export const imageUploadStore = {
+    mutations: {
+        SET_IMAGEPATH(state, response) {
+            if(response.code == 500) {
+                this.state.upload_message = 'Error uploading the file, please try again'
+                alert(this.state.upload_message)
+            }
+           
+            if(response.code == 200) {
+                this.state.imagePath = newState.data.imagePath
+            }
+
+            else {
```

Comment:
this applies to when this check returns true `code != 200` reduce this entire thing to only one if and else block